sympy/sets/fancysets.py

```diff
@@ -40,6 +40,7 @@ class Rationals(with_metaclass(Singleton, Set)):
     _inf = S.NegativeInfinity
     _sup = S.Infinity
     is_empty = False
+    is_finiteset = False
```

Comment:
I have named this as `is_finite_set` in #16764 and I thought that it could be more accurately translating `FiniteSet` into snakecase.

Reply:
That seems reasonable I've changed to `is_finite_set`